<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1358</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1181</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mainLayout" stretch="2,4,1">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <layout class="QGridLayout" name="clipsLayout">
       <item row="4" column="0">
        <widget class="QLineEdit" name="baseFolderEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="ThumbListWidget" name="clipsList"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Base Clips Folder</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Clips</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="moveFirstPushButton">
           <property name="text">
            <string>Move first</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="moveLastPushButton">
           <property name="text">
            <string>Move last</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="beforePrevPushButton">
           <property name="text">
            <string>Before prev</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="afterNextPushButton">
           <property name="text">
            <string>After next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="videoLayout" stretch="4,1">
       <item>
        <widget class="VideoWidget" name="clipWidget" native="true"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="1,3,5">
         <item row="2" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Volume</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="volumeSlider">
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Clip range</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="videoTime">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="keepClip">
           <property name="text">
            <string>Keep clip</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QRangeSlider" name="rangeSlider" native="true"/>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>Delete Video</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="keepOnNext">
           <property name="text">
            <string>Keep on next video</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="infoLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="clipsCountLabel">
         <property name="text">
          <string>Clips:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="finalDurationLabel">
         <property name="text">
          <string>Final duration: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="durationLabel">
         <property name="text">
          <string>Duration: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="viewerCount">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Views:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="subtitleGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>110</height>
          </size>
         </property>
         <property name="title">
          <string>Subtitles Alignment</string>
         </property>
         <widget class="QRadioButton" name="subAlignRadioButton_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">subsAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="subAlignRadioButton_8">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">subsAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="subAlignRadioButton_9">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">subsAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="subAlignRadioButton_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>16</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">subsAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="subAlignRadioButton_5">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">subsAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="subAlignRadioButton_6">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>40</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">subsAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="subAlignRadioButton_1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">subsAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="subAlignRadioButton_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>60</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">subsAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="subAlignRadioButton_3">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>60</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">subsAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="subAlignRadioButton_0">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>80</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(239, 41, 41);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">subsAllignmenButtonGroup</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="titleGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>110</height>
          </size>
         </property>
         <property name="title">
          <string>Title Alignment</string>
         </property>
         <widget class="QRadioButton" name="titleAlignRadioButton_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">titleAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="titleAlignRadioButton_8">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">titleAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="titleAlignRadioButton_9">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">titleAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="titleAlignRadioButton_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>16</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">titleAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="titleAlignRadioButton_5">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">titleAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="titleAlignRadioButton_6">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>40</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">titleAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="titleAlignRadioButton_1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">titleAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="titleAlignRadioButton_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>60</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">titleAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="titleAlignRadioButton_3">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>60</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">titleAllignmenButtonGroup</string>
          </attribute>
         </widget>
         <widget class="QRadioButton" name="titleAlignRadioButton_0">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>80</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(239, 41, 41);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">titleAllignmenButtonGroup</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="intervalCheckBox">
         <property name="text">
          <string>Apply transition after</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="titleTextEdit"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1358</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open Clips Json...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ThumbListWidget</class>
   <extends>QListWidget</extends>
   <header>clips_editor.widgets.list_widgets.thumb_list_widget</header>
  </customwidget>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>clips_editor.widgets.video_widgets.video_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QRangeSlider</class>
   <extends>QWidget</extends>
   <header>clips_editor.widgets.range_slider.range_slider</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="titleAllignmenButtonGroup"/>
  <buttongroup name="subsAllignmenButtonGroup"/>
 </buttongroups>
</ui>
